pulsar/client_impl_test.go

```diff
@@ -425,7 +426,7 @@ func TestNamespaceTopics(t *testing.T) {
 		t.Fatal(err)
 	}
 	topic2 := fmt.Sprintf("%s/topic-2", namespace)
-	if err := httpPut("admin/v2/persistent/"+topic2, namespace); err != nil {
```

Comment:
Could you explain changes in this test?

Reply:
This request got a 400 response after enabling TLS.
I confirmed to @freeznet  it is an error code.
So I change its logic to be like the logic of topic 1.